java/dataset/src/main/java/org/apache/arrow/dataset/jni/JniWrapper.java

```diff
@@ -52,7 +52,7 @@ private JniWrapper() {
    * Create Dataset from a DatasetFactory and get the native pointer of the Dataset.
    *
    * @param datasetFactoryId the native pointer of the arrow::dataset::DatasetFactory instance.
-   * @param schema the predefined schema of the resulting Dataset.
+   * @param schema           the predefined schema of the resulting Dataset.
```

Comment:
nit: spaces are not necessary

Reply:
Thanks! But I don't see we always make consistent practices of Javadoc alignment in Arrow Java code. E.g.

https://github.com/apache/arrow/blob/78c88a9f517b540ea010a6bd6a8c8f6d91769559/java/gandiva/src/main/java/org/apache/arrow/gandiva/evaluator/JniWrapper.java#L34-L40

Maybe what we need here is to add a relevant rule to checkstyle config and do a complete clean up?